trinity/server.py

```diff
@@ -163,6 +178,17 @@ def _make_peer_pool(self) -> PeerPool:
         self.logger.info("Closing server...")
         await self._close_tcp_listener()
 
+    def _get_discv5_topic(self) -> bytes:
```

Comment:
This looks like a good candidate to 

- extract to a utility function (probably passing in the `genesis_hash` as an argument) 
- write explicit tests against the `LESProtocol` classes and known genesis blocks (ropsten/mainnet) that we support.

Reply:
Indeed, I've done that